modules/runner/src/testFixtures/java/org/apache/ignite/internal/ClusterPerClassIntegrationTest.java

```diff
@@ -185,6 +185,42 @@ protected static void insertPersons(String tableName, Person... people) {
         );
     }
 
+    /**
+     * Updates data in the table created by {@link #createZoneAndTable(String, String, int, int)}.
+     *
+     * @param tableName Table name.
+     * @param people People to update in the table.
+     */
+    protected static void updatePersons(String tableName, Person... people) {
```

Comment:
Nitpicking: plural form of 'person' seems to be 'people'

Reply:
Why not